pyproject.toml

```
@@ -1,40 +1,13 @@
-[tool.pysen]
-version = "0.11.0"
+[tool.ruff]
+target-version = "py311"
 
-[tool.pysen.lint]
-enable_black = true
-enable_flake8 = true
-enable_isort = true
-enable_mypy = true
-mypy_preset = "strict"
-mypy_plugins = [
-  { function = "numpy.typing.mypy_plugin" },
-  { function = "pydantic.mypy" },
-]
-line_length = 88
-py_version = "py311"
-isort_known_first_party = ["voicevox_engine"]
-isort_known_third_party = ["numpy"]
-[[tool.pysen.lint.mypy_targets]]
-paths = ["."]
+[tool.ruff.lint]
+select = ["E4", "E7", "E9", "F", "B", "I"]
+ignore = ["B9"]
```

> B900番台を有効にするか

個人的には賛成です！！
このPRではignoreするのも賛成です！！
ここにTODO書いておくと良いかも？　雰囲気こんな感じの･･･！
```suggestion
ignore = ["B9"] # TODO: 有効化する
```

---

ありがとうございます、下記の件とあわせてignoreにTODOコメントをつけました
https://github.com/VOICEVOX/voicevox_engine/pull/1563/commits/0ca291310e15f20e6e8ce6f6733b5e460dfefbe0